facebook_connector.py

```diff
@@ -0,0 +1,110 @@
+"""
+
+Author: Nigel Schuster <nigel schusters at gmail dot com>
+
+This module provides a simple abstraction for accessing the Facebook chat data
```

Comment:
Change this to "This module provides a simple abstraction for accessing Facebook messages data from a downloaded archive from facebook.com"